srv/salt/ceph/mds/files/keyring.j2

```diff
@@ -1,6 +1,7 @@
 [{{ client }}]
         key = {{ secret }}
-        caps mds = "allow"
-        caps mon = "allow profile mds"
-        caps osd = "allow rwx"
+        caps mds = "allow *"                         
```

Comment:
Well, it doesn't exist if the provided keyring *exactly* matches the previous keyring(including the *)